client/src/components/ImageCard/ImageCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./ImageCard.css";
+
+// Function based React Component
+const ImageCard = props => {
+  // Default Class to apply to Component
+  let classList = `ImageCard`;
+  let image = <img className="backgroundImage" src={props.url} alt="trip background" />;
+  let textEls = props.text.map((text, index) => {
+    if (index === 0) {
+      return (
+        <h3 className="first" key={index}>
+          {text}
+        </h3>
+      );
+    }
+    return (
+      <h3 className="textEl" key={index}>
+        {text}
+      </h3>
+    );
+  });
+  return (
+    <div className={classList}>
```

Comment:
If you're not changing or updating this class, can you just reference the class as a string here, instead of using this variable?